tests/scripts/utils.sh

```diff
@@ -100,13 +85,15 @@ function run_suite() {
   
   install_ols "$1" "$3" "$4" "$5" "$6" "$7" "$8" "$9"
 
-  # wait for the ols api server to come up
-  wait_for_ols "$OLS_URL"
-  if [ $? -ne 0 ]; then
-    echo "Timed out waiting for OLS to become available"
-    echo "${JUNIT_TEMPLATE}" | sed "s/SUITE_ID/${1}/g" > $ARTIFACT_DIR/junit_setup_${1}.xml
+ # Determine the hostname for the ols route
+  export OLS_URL=https://$(oc get route ols -o jsonpath='{.spec.host}')
+  echo "OLS_URL set to $OLS_URL"
```

Comment:
to be fair this PR does what the ticket asked, but would it be terribly difficult to also move this logic into the setup while you're doing the refactoring?